doc/adr/0006-helmfile-base-values.md

```diff
@@ -0,0 +1,83 @@
+# 6) Production as the configuration base case for helmfile values {#adr_0006}
+
+Date: 2023-01-17
+
+## Status
+
+accepted
+
+## Context
+
+We deploy Wikibase Cloud to different environments, i.e. `production` and `staging`.
+It's also possible to run the setup on dev machines using the `local` environment.
+Configuration values across these environments has been heavily duplicated.
+This makes it easy to miss updates for a certain environment, or inadvertently deploying changes that weren't meant to be deployed to a certain environment.
+
+This is why we've been looking for ways to DRY these value files up.
+
+While the [most straight forward way][helmfile-docs] of de-duplicating values is easy to do, it creates one-off hierarchy patterns for each helmfile release, and every change to be done requires understanding yet another ad-hoc solution.
+
+
+Instead, we'd like to end up in a situation where every helmfile release uses the same pattern, making it obvious how and where to apply pending changes.
+
+## Decision
+
+Instead of defining [a least common set of values for a service in a `base` case][helmfile-docs], and then apply per-environment changes on top of this base case, __we want to always use the values for production as the base case__ and then define overrides per staging and production.
```

Comment:
```suggestion
Instead of defining [a common set of values for a service in a `base` case][helmfile-docs], and then apply per-environment changes on top of this base case, __we want to always use the values for production as the base case__ and then define overrides per staging and production.
```

minor nitpick to perhaps clarify the meaning

Reply:
This was meant to say "aims at making it as minimal as possible with no regards for other aspects such as easy to understand et. al." Is there a better way to say that or do you just go "common"?

Reply:
I can't think of a nicer phrasing than common; perhaps "naively common"?

"least common" to me however implies "values that had the fewest number of things in common" which is really not what we mean.

Reply:
Then let's just do "common" I guess.